src/pages/MenuPage.js

```diff
@@ -33,14 +26,20 @@ export class MenuPage extends React.Component {
   componentWillMount() {
     this.databaseListenerId = this.props.database.addListener(
       // Ensure that language changes in login modal are re-rendered onto the MenuPage
-      (changeType, recordType) => recordType === 'Setting' && this.forceUpdate()
+      (changeType, recordType) => recordType === 'Setting' && this.forceUpdate(),
     );
   }
 
   componentWillUnmount() {
     this.props.database.removeListener(this.databaseListenerId);
   }
 
+  exportData = () => {
+    const { settings, database } = this.props;
+    const syncSiteName = settings.get(SYNC_SITE_NAME);
```

Comment:
The settings could easily come from within the exportData method in UIDatabase, I should probably do that

Reply:
Actually I won't accessing settings is a bit janky in UIDatabase. Ugh.